src/quo/components/wallet/transaction_progress/schema.cljs

```diff
@@ -2,7 +2,8 @@
 
 (def ^:private ?network
   [:map
-   [:network {:optional true} [:maybe [:enum :mainnet :optimism :arbitrum]]]
+   [:network {:optional true} [:maybe :keyword]]
```

Comment:
Can we use `:schema.quo/networks` here?

Reply:
initially I was, but the usage of this component also passes combinations of networks for some reason e.g. `:mainnet-optimism` or `:optimism-arbitrum`. This component isn't actually used anywhere tho, except for the preview.